docs/.vuepress/public/presets/toml/pastel-powerline.toml

```
@@ -72,7 +72,7 @@ format = '[ $symbol ($version) ]($style)'
 [docker_context]
 symbol = " "
 style = "bg:#06969A"
-format = '[ $symbol $context ]($style) $path'
+format = '[ $symbol $context ]($style)$path'
```

LGTM, but please also remove `$path`, there is no such variable.
```suggestion
format = '[ $symbol $context ]($style)'
```